engine/runtime/src/bench/scala/org/enso/interpreter/bench/fixtures/semantic/AtomFixtures.scala

```diff
@@ -105,9 +104,9 @@ class AtomFixtures extends DefaultInterpreterRunner {
   val sumListMethodsCode =
     """from Standard.Base.Data.List import all
       |
-      |Nil.sum self = acc -> acc
-      |Cons.sum self = acc -> case self of
+      |List.List.sum self acc = case self of
       |    Cons h t -> @Tail_Call t.sum h+acc
+      |    _ -> acc
```

Comment:
```suggestion
      |    Nil -> acc
```
That's just my opinion and it's not a very important piece of code, but why use fallback instead of the only other acceptable constructor?

Reply:
Until we have at least a rudimentary typechecker, `_` is slightly faster than your proposal, because we still need to check the scrutinee if it misses the first branch. In the future, your code should optimize to the `_` anyway.